network/sites.csv

```diff
@@ -862,6 +862,7 @@ EWZ,10,-43.510031722,170.852603355,650,,WGS84,Unknown,1991-03-21T03:43:00Z,2001-
 EXPV,10,-39.253545373,175.564881983,2030,,WGS84,Unknown,1995-10-25T03:58:00Z,1995-11-24T06:22:00Z
 EYCZ,10,-43.42136,172.35544,149.7,,WGS84,External GPS Device,2012-03-20T21:00:01Z,2013-12-02T22:00:00Z
 EYRS,20,-43.42138,172.35544,119,,WGS84,External GPS Device,2011-03-25T05:00:00Z,2012-10-15T10:45:00Z
+EY2M,50,-43.47384,172.39266,88,,WGS84,Unknown,2024-04-04T16:00:00Z,9999-01-01T00:00:00Z
```

Comment:
- please check the height here, need to be consistent with the info in stations.csv

Reply:
This is the correct value